step-templates/iis-website-set-ip-security.json

```diff
@@ -3,13 +3,13 @@
   "Name": "IIS Website - Set IP Security",
   "Description": "Takes a list of ip/mask and allow them in ipsecurity",
   "ActionType": "Octopus.Script",
-  "Version": 8,
+  "Version": 9,
   "CommunityActionTemplateId": null,
   "Properties": {
     "Octopus.Action.Script.Syntax": "PowerShell",
     "Octopus.Action.Script.ScriptSource": "Inline",
     "Octopus.Action.RunOnServer": "false",
-    "Octopus.Action.Script.ScriptBody": "function allowIp([string]$Website, [string]$ip, [string]$mask) {\r\n    Write-Output \"Adding ip $ip/$mask to allow\"\r\n    add-webconfiguration /system.webServer/security/ipSecurity -location $Website -value @{ipAddress=\"$ip\";subnetMask=\"$mask\";allowed=\"true\"} -pspath IIS://\r\n}\r\n\r\nClear-WebConfiguration -Filter /system.webServer/security/ipSecurity -location $Site\r\n\r\nif ($EnableProxyMode -eq \"true\") {\r\n    Write-Output \"Enabling proxy mode\"\r\n    #set-webconfigurationproperty -Filter /system.webServer/security/dynamicIpSecurity -location $Site -Name \"enableProxyMode\" -Value \"true\"\r\n    set-webconfigurationproperty -Filter /system.webServer/security/ipSecurity -location $Site -Name \"enableProxyMode\" -Value \"true\"\r\n}\r\n\r\nif ($SetDeny -eq \"true\") {\r\n    Write-Output \"Setting Deny rule\"\r\n    set-webconfigurationproperty -Filter /system.webServer/security/ipSecurity -location $Site -Name \"allowUnlisted\" -Value \"false\"\r\n}\r\n\r\n$ips = $IpAddresses -split '\\n'\r\n$ips = $ips.Trim('\\r')\r\nforeach ($u in $ips) {\r\n    $a = $u.Split(\"/\")\r\n    allowIp $Site $a[0] $a[1]\r\n}\r\n",
+    "Octopus.Action.Script.ScriptBody": "function allowIp([string]$Website, [string]$ip, [string]$mask) {\r\n    $value = @{ipAddress=\"$ip\";allowed=\"true\"}\r\n    if ($mask -ne \"\")\r\n    {\r\n        Write-Output \"Adding ip $ip/$mask to allow\"\r\n        $value.subnetMask = $mask\r\n    }\r\n    else\r\n    {\r\n         Write-Output \"Adding ip $ip to allow\"\r\n    }\r\n\r\n    add-webconfiguration /system.webServer/security/ipSecurity -location $Website -value $value -pspath IIS://\r\n}\r\n\r\nClear-WebConfiguration -Filter /system.webServer/security/ipSecurity -location $Site\r\n\r\nif ($EnableProxyMode -eq \"true\") {\r\n    Write-Output \"Enabling proxy mode\"\r\n    #set-webconfigurationproperty -Filter /system.webServer/security/dynamicIpSecurity -location $Site -Name \"enableProxyMode\" -Value \"true\"\r\n    set-webconfigurationproperty -Filter /system.webServer/security/ipSecurity -location $Site -Name \"enableProxyMode\" -Value \"true\"\r\n}\r\n\r\nif ($SetDeny -eq \"true\") {\r\n    Write-Output \"Setting Deny rule\"\r\n    set-webconfigurationproperty -Filter /system.webServer/security/ipSecurity -location $Site -Name \"allowUnlisted\" -Value \"false\"\r\n}\r\n\r\n$ips = $IpAddresses -split '\\r\n'\r\n$ips = $ips.Trim('\\r')\r\nforeach ($u in $ips) {\r\n    $a = $u.Split(\"/\")\r\n    allowIp $Site $a[0] $a[1]\r\n}",
```

Comment:
This change was actually an unintended, I didn't mean to add the `\\r`.  As such I could either revert to `$ips = $IpAddresses -split '\\n'` or I could correct to `$ips = $IpAddresses -split '\\r\\n'`.
Which do you recommend?